sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/_input_output_helpers.py

```diff
@@ -241,6 +244,8 @@ def to_rest_dataset_literal_inputs(
                 # set mode attribute manually for binding job input
                 if "mode" in input_value:
                     input_data.mode = input_value["mode"]
+                if "pathOnCompute" in input_value:
```

Comment:
why literal input has path_on_compute setting?

Reply:
yes literal input does not need this

Reply:
fixed it